### ISSUE TITLE
org-jira-update-issue fails with "No project with the provided key exists"

### ISSUE THREAD

**Issue by mruediger** [ahungry/org-jira#235]
Hey there.

When ever I run `org-jira-update-issue` I am getting the following error message:

```
About to sync worklog for issue: DSP-6752 in file: DSP
jiralib-get-user-account-id: Wrong type argument: listp, (errorMessages . ["No project with the provided key exists"])
```
Can anyone help me on debugging this issue? I am pretty new to `emacs` and `org-jira` and don't know were to start but would be very grateful if someone would point me into the right direction.

**Comment by AblatedSprocket** [comment]
I'm seeing the same thing.

**Comment by ahungry** [comment]
ffdb3e39 appears to have introduced the function jiralib-get-user-account-id, which was from a PR I accepted having to do with getting account ids - I'm not sure if this is related.

**Comment by ahungry** [comment]
Yes, that seems to be the commit that caused the issue.  The problem is that jiralib-get-user-account-id is passing nil instead of a valid project-key when calling jiralib-get-users, and that is not a valid input.

It likely was not discovered previously, as other actions will cause jiralib-get-users to be called *with* a valid project key, in which case the list of users becomes cached in jiralib-users-cache and the invalid call with nil is not performed.

**Comment by ahungry** [comment]
Fixed in df45be1
